### ISSUE TITLE
Please improve structure and getting started guide

### ISSUE THREAD

**Issue by kristianmandrup** [cmichaelgraham/aurelia-typescript#21]
Been reading your blog post and was excited to get started with Aurealia and TS, however...

I wish you would split this repo into more repos, at least group them in some logical fashion: one for Visual Studio, One for skeleton app etc. How would I start a skeleton project using Sublime Text 3?
Looks like there is also specific support for Atom IDE. How about Webstorm?

At least make it more clear in the README, perhaps move the Visual Studio specifics at least to the Wiki for now? Then have a Geting started section for each Editor/IDE supporting TypeScript ;)
I would be glad to help out... Cheers!

**Comment by cmichaelgraham** [comment]
i'm thinking about creating a skeleton-navigation style app and porting it to various ide's - here's the list i'm considering:

the naming convention would be: aurelia-typescript-<ide name>

1. atom / atom-typescript [`aurelia-typescript-atom`](https://github.com/cmichaelgraham/aurelia-typescript-atom)
2. webstorm
3. sublime text
4. gulp-typescript + any editor (notepad++)
3. vs 2013
4. vs 2015
5. vs code

**Comment by kristianmandrup** [comment]
Great, I like it! Just make a hello world app, such as skeleton app without the Flickr and child router routes.
Why not create a separate repo for each such app that you listed? Or is it a maintenance issue and reuse potential which is the problem? In that case, perhaps use a Yeoman generator or similar such as slush.

Exciting times!!! Hope others will join your amazing effort... Typescript is the way ahead :)

**Comment by veikkoeeva** [comment]
Indeed. Naturally as complete as possible example what comes to "plumbing". Like non-minimized and non-bundled in debug setting, but on release doing it all to make things run fast. With TypeScript, VS 2015 to a ASP.NET 5 template calling WebApi and a Web Deploy setup "to somewhere" would be really cool. I think Aurelia itself is a joy to work with, but setting up a real application might be a bit of a trouble with the aforementioned technology stack. I think even better would be to have amble comments and documentation on why and what the pieces there are (even if they might be "clear" otherwise) just to facilitate the learning process for those new to web programming.

A lot of wished for the hearts and minds, I understand. But a VS example would be really nice. :) I just wanted to write this here, though you are doing a great job already!

**Comment by kristianmandrup** [comment]
I can help write a generator which adds configuration depending on the editor chosen
